observer.py

```diff
@@ -207,10 +222,9 @@ async def run_coingecko_get_price():
         help="Prometheus Exporter port",
     )
     parser.add_argument(
-        "-i",
         "--ignore",
         nargs="+",
-        help="List of symbols to ignore",
+        help="List of symbols and/or events to ignore",
```

Comment:
Explain how the argument works. Give some examples like `Crypto.BTCUSD/...` so it's clear to the user. E.g., `Crypto.BTCUSD/.*` matches all alerts on BTCUSD.